_episodes/03-create.md

```diff
@@ -273,6 +273,12 @@ $ pwd
 
 ~~~
 $ nano draft.txt
+~~~
+{: .language-bash}
+
+Let's again type in a few lines of text. Once we're happy with our text, we can press `Ctrl-O` to write our data to disk. Once our file is saved, we can use `Ctrl-X` to quit the editor and return to the shell.
```

Comment:
Is there a step missing here? Nano will prompt to confirm the file name when saving, which requires the user to press *enter*, which is different to the exit command.